app/models/user.rb

```diff
@@ -82,7 +82,11 @@ def client_admin?
   end
 
   def admin?
-    self.has_role?('admin')
+    has_role?('admin')
+  end
+
+  def not_admin?
+    !admin?
   end
```

Comment:
I do prefer readability over concision. I don't usually describe Ruby as terse, although compared to something in Obj-C it certainly is.

I think prepending a method with `!` can be very readable at times, but I tend to make small helper methods for `!`-prepended methods because it's just more pleasant to read. Plus, it makes it much different from something like `!!admin?` which is probable my least favorite thing in Ruby and something that makes me cry every time I see it.